localstack/services/cloudformation/service_models.py

```diff
@@ -92,10 +89,15 @@ def get_cfn_attribute(self, attribute_name):
         """Retrieve the given CF attribute for this resource"""
         return self.props.get(attribute_name)
 
+    # TODO: make this stricter
+    def get_ref(self):
+        return self.physical_resource_id or self.get_physical_resource_id()
```

Comment:
I think this relationship still needs exploring. It's probably the right thing to do for now, but I would like to know if the physical resource id is always what's returned when using the `Fn::Ref` intrinsic.